src/locales/ar_IQ/translations.json

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
are translations files supposed to be committed? I thought they were generated at build time.

Reply:
Good catch.
I need to update `.gitignore` because it still references those in `packages/[app|plugin]`.